druid/src/widget/sized_box.rs

```diff
@@ -45,6 +45,10 @@ impl<T> SizedBox<T> {
     }
 
     /// Construct container without child, and both width and height not set.
+    ///
+    /// If the widget is unchanged, it will do nothing, which can be useful if you want to draw a
+    /// widget some of the time (for example, it is used to implement
+    /// [`Maybe`][crate::widget::Maybe]).
     pub fn empty() -> Self {
```

Comment:
adding `#[doc(alias = "null")]` will help discoverability

Reply:
Ack I was genuinely going to do that but forgot. Thanks!